quattor/types/system.pan

```diff
@@ -27,7 +27,7 @@ type structure_oldname = {
 include 'components/network/core-schema';
 
 # TODO: monitoring to be defined
-include 'monitoring/lemon/schema';
+include { if_exists('monitoring/lemon/schema') };
```

Comment:
I don't think that `if_exists` is a good idea either, I just think that sites who need it can add `include 'monitoring/lemon/schema'; ` in their site config and that `monitoring/lemon/schema` should do the type binding.